app/views/reasons/ChangeUnderpaymentReasonView.scala.html

```diff
@@ -20,14 +20,14 @@
         linkButton: components.linkButton
 )
 
-@(backLink: Call, summaryList: SummaryList, boxNumber: Int)(implicit request: Request[_], messages: Messages)
+@(backLink: Call, summaryList: SummaryList, heading: String)(implicit request: Request[_], messages: Messages)
 
 @layout(
-    pageTitle = messages("changeUnderpaymentReason.heading", boxNumber),
+    pageTitle = heading,
```

Comment:
why is heading page title, we usually have two messages one for title one for heading even though it's the same